notebooks/official/automl/sdk_automl_forecasting_hierarchical_batch.ipynb

```diff
@@ -61,7 +66,7 @@
         "## Overview\n",
         "\n",
         "\n",
-        "This tutorial demonstrates how to use the Vertex AI SDK for Python to create hierarchical forecasting models using a Google Cloud [AutoML](https://cloud.google.com/vertex-ai/docs/start/automl-users)and do batch prediction. Specifically, you predict a fictional store's sales based on historical sales data.\n",
+        "This tutorial demonstrates how to use the Vertex AI SDK for Python to create hierarchical forecasting models using  Google Cloud [AutoML](https://cloud.google.com/vertex-ai/docs/start/automl-users) and do batch prediction. Specifically, you predict a fictional store's sales based on historical sales data.\n",
```

Comment:
Regarding: "This tutorial demonstrates how to use the Vertex AI SDK for Python to create hierarchical forecasting models using  Google Cloud [AutoML](https://cloud.google.com/vertex-ai/docs/start/automl-users) and do batch prediction." The link goes to a page in Vertex AI documentation that deals with the difference between legacy AutoML and  Vertex AI AutoML. This does not make sense. Legacy AutoML is completely deprecated. Also, should "Google Cloud AutoML" be "Vertex AI AutoML"? This notebook looks questionable.

Reply:
@kittyabs , I made a few changes in terms of the verbiage in the markdown cells. Can you please check this once again to see if the modified verbiage is in line with the corresponding documentation present at [link](https://cloud.google.com/vertex-ai/docs/tabular-data/forecasting/hierarchical)

Reply:
Row/cell 69 is unchanged. Where, exactly, are your new edits?

Reply:
The changes made are very minimal. As mentioned, that the "AutoML" link goes to a page in Vertex AI documentation that deals with the difference between legacy AutoML and Vertex AI AutoML, this is now change to "Vertex AI" with no link as automl is now vertexai automl. 

I verified the verbiage in the notebook is in line with the documentation present at [hierarchical forecasting](https://cloud.google.com/vertex-ai/docs/tabular-data/forecasting/hierarchical)

A potential change could be the title of the notebook "Vertex AI SDK for Python: AutoML training hierarchical forecasting for batch prediction" which can be changed to "Vertex AI SDK for Python: VertexAI AutoML training hierarchical forecasting for batch prediction".

Please let me know if the notebook still feel questionable and what could be the next action items on this notebook

Reply:
Sure, go ahead and make the change

Reply:
made the title change. Please review